### DISCUSSION TITLE
Apple Home App - Bosch notifications

### DISCUSSION THREAD

**Discussion by cohen-sydney** [thoukydides/homebridge-homeconnect#158] [General]
I got this working with HomeBridge and from that to Apple Home App.
The Home app popos up notifications onky for this App. when some things is done to the dishwasher. is there aw ay to stop these pop ups.

**Comment by thoukydides** [comment]
Are you referring to the door open/close notifications? If so, then those can be disabled within the Apple Home app.

This was discussed in #132.

**Reply by cohen-sydney** [reply]
Bosch dishwasher but yes. i did look but I could not see how to do that but that discussion showed me how.

**Reply by cohen-sydney** [reply]
thanks
